backend/src/main/java/pl/edu/agh/apdvbackend/mappers/group/admin_panel/AdminPanelGroupFieldMapper.java

```diff
@@ -0,0 +1,30 @@
+package pl.edu.agh.apdvbackend.mappers.group.admin_panel;
+
+import java.util.List;
+import org.mapstruct.Mapper;
+import org.mapstruct.Mapping;
+import pl.edu.agh.apdvbackend.models.body_models.group.AdminPanelGroupResponseBody;
+import pl.edu.agh.apdvbackend.models.database.Field;
+
+@Mapper(componentModel = "spring")
+public abstract class AdminPanelGroupFieldMapper {
+
+    public List<AdminPanelGroupResponseBody.Endpoint.Field> toFieldList(
+            List<Field> fields,
+            List<Field> enableFields) {
+        return fields
+                .stream()
+                .map(field -> toField(field, enableFields))
+                .toList();
+    }
+
+    @Mapping(target = "unitName", source = "field.unit.name")
+    @Mapping(target = "isBelongingToGroup", expression = "java(isFieldBelongsToGroup(field, enableFields))")
+    public abstract AdminPanelGroupResponseBody.Endpoint.Field toField(
+            Field field, List<Field> enableFields);
+
+    protected boolean isFieldBelongsToGroup(Field field,
```

Comment:
plox zmień nazwę tej metody

Reply:
Hmmm, nwm, czy danie `belongsToGroup` daje nam wystarczający kontekst, co ta funkcja ma robić. Imo słówko `field` powinno się pojawić w nazwie, żeby dać znać, co konkretnie chcemy sprawdzać, skoro funkcja ma dwa parametry

Reply:
to doesFieldBelongToGroup albo isFieldInGroup

Reply:
Zmieniłem na `isFieldInGroup`